packages/wonder-blocks-typography/index.js

```diff
@@ -18,6 +18,45 @@ import Tagline from "./components/tagline.js";
 import Caption from "./components/caption.js";
 import Footnote from "./components/footnote.js";
 
+/**
+ * Typography components for headings or titles.
+ */
+export type Heading =
+    | typeof Title
+    | typeof HeadingLarge
+    | typeof HeadingMedium
+    | typeof HeadingSmall
+    | typeof HeadingXSmall;
+
+/**
+ * Typography components for representing body text.
+ */
+export type BodyText =
+    | typeof Body
+    | typeof BodySerif
+    | typeof BodySerifBlock
+    | typeof BodyMonospace;
+
+/**
+ * Typography components for labels.
+ */
+export type Label =
+    | typeof LabelLarge
+    | typeof LabelMedium
+    | typeof LabelSmall
+    | typeof LabelXSmall;
+
+/**
+ * All typography components.
+ */
+export type Typography =
+    | Heading
+    | BodyText
+    | Label
+    | typeof Tagline
+    | typeof Caption
+    | typeof Footnote;
```

Comment:
Couldn't put these types into the `types.js` as that would create a cyclic dependency among the components. Should we have an `external-types.js` file or something? Or are these ok here? I like them here since it means that if we add a new component, it's clear these need updating too.

Reply:
These are okay here.  Thanks for adding these groupings.